_posts/2016-07-23-reopening-instructor-training.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+subheadline: "Two open instructor training classes this fall"
+title: "Reopening Instructor Training"
+teaser: ""
+header:
+   image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+   - blog
+comments: true
+show_meta: true
+authors: ["Greg Wilson"]
+---
+
+For the last ten months,
+the [Software Carpentry Foundation](http://software-carpentry.org/scf/) has worked toward three goals
+for its instructor training program:
+
+1.  Make the content more relevant.
+
+2.  Increase the number of people able to deliver instructor training.
+
+3.  Find a format that meets everyone's needs in a sustainable way.
+
+They have made a lot of progress on all three,
+and are therefore now able to offer instructor training once again
+to people who aren't affiliated with our partner organizations.
+If you wish to apply to take part
+in one of the two open-enrollment classes we will offer this fall,
+please fill in the form at:
+
+<https://amy.software-carpentry.org/workshops/request_training/>
+
+to tell them about yourself,
+what excites you about teaching,
+and how Software and Data Carpentry can help in your community.
+They will notify applicants as spaces become available.
+If you have any questions,
+please mail
+[training@software-carpentry.org](mailto:training@software-carpentry.org).
+
+Please note that as a condition of taking this training, you must:
+
+*   abide by our code of conduct, which can be found at
+    <http://datacarpentry.org/code-of-conduct/> and
+    <http://software-carpentry.org/conduct/>,
+
+*   agree to teach at a Data Carpentry or Software Carpentry workshop
+    within 12 months of the course,
+    and
+
+*   complete three short tasks after the course in order to complete certification.
+    The tasks take a total of approximately 8-10 hours,
+    and are described at
+    <http://swcarpentry.github.io/instructor-training/checkout/>.
+
+If you would like to accelerate the process,
```

Comment:
Because this immediately follows the description of 8-10 hour checkout tasks, it can be read as accelerating that  checkout process and so this may be confusing.